<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="43554" started-at="2018-08-10T17:09:15Z" finished-at="2018-08-10T17:09:58Z">
    <groups>
      <group name="smoke">
        <method signature="TwoLinksAssertion.ClickOnTwoLinks()[pri:0, instance:com.healthdata.tests.TwoLinksAssertion@31ef45e3]" name="ClickOnTwoLinks" class="com.healthdata.tests.TwoLinksAssertion"/>
      </group> <!-- smoke -->
    </groups>
    <test name="Default test" duration-ms="43554" started-at="2018-08-10T17:09:15Z" finished-at="2018-08-10T17:09:58Z">
      <class name="com.healthdata.tests.TwoLinksAssertion">
        <test-method status="PASS" signature="setUpTest()[pri:0, instance:com.healthdata.tests.TwoLinksAssertion@31ef45e3]" name="setUpTest" is-config="true" duration-ms="701" started-at="2018-08-10T17:09:14Z" finished-at="2018-08-10T17:09:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpTest -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.healthdata.tests.TwoLinksAssertion@31ef45e3]" name="setUp" is-config="true" duration-ms="34408" started-at="2018-08-10T17:09:15Z" finished-at="2018-08-10T17:09:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="ClickOnTwoLinks()[pri:0, instance:com.healthdata.tests.TwoLinksAssertion@31ef45e3]" name="ClickOnTwoLinks" duration-ms="7791" started-at="2018-08-10T17:09:49Z" finished-at="2018-08-10T17:09:57Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Results does match expected [null] but found [55 results]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Results does match expected [null] but found [55 results]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.healthdata.tests.TwoLinksAssertion.ClickOnTwoLinks(TwoLinksAssertion.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ClickOnTwoLinks -->
        <test-method status="FAIL" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.healthdata.tests.TwoLinksAssertion@31ef45e3]" name="tearDown" is-config="true" duration-ms="1290" started-at="2018-08-10T17:09:57Z" finished-at="2018-08-10T17:09:58Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=ClickOnTwoLinks status=FAILURE method=TwoLinksAssertion.ClickOnTwoLinks()[pri:0, instance:com.healthdata.tests.TwoLinksAssertion@31ef45e3] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.healthdata.tests.TestBase.tearDown(TestBase.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDownTest()[pri:0, instance:com.healthdata.tests.TwoLinksAssertion@31ef45e3]" name="tearDownTest" is-config="true" duration-ms="3" started-at="2018-08-10T17:09:58Z" finished-at="2018-08-10T17:09:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownTest -->
      </class> <!-- com.healthdata.tests.TwoLinksAssertion -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
